### DISCUSSION TITLE
Is it possible to provide devtool for vue3

### DISCUSSION THREAD

**Discussion by chaozwn** [TanStack/query#4417] [Q&A]
Do you have an adapter for vue devtool？

**Comment by DamianOsipiuk** [comment]
The only thing you need to do is to install official [Vue Devtools](https://devtools.vuejs.org/guide/installation.html)

Vue Query will seemingly integrate with official devtools, adding custom inspector and timeline events. Devtools would be treeshaken from production bundles by default.
